src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml

```diff
@@ -169,5 +170,30 @@ limitations under the License.
       <access op="invoke" privilege="administer"/>
       <mandatoryConform/>
     </command>
+
+    <command source="client" code="0x0c" name="SetVidVerificationStatement" isFabricScoped="true" optional="false" apiMaturity="provisional">
+      <description>This command SHALL be used to update any of the accessing fabric's associated VendorID, VidVerificatioNStatement or VVSC (Vendor Verification Signing Certificate).</description>
+      <arg id="0" name="VendorID" type="int16u" optional="true"/>
+      <arg id="1" name="VidVerificationStatement" type="octet_string" optional="true"/>
```

Comment:
I think these might need lengths. Not sure what that's used for though, but others seem to have it.

Reply:
Lengths will be validated by code. Adding it here actually does not do what we think as it doesn't validate lengths during decode, and is only used for RAM attributes allocation.